WordPress/src/main/java/org/wordpress/android/ui/media/MediaItemFragment.java

```diff
@@ -416,9 +414,4 @@ private void copyUrlToClipboard() {
             ToastUtils.showToast(getActivity(), R.string.error_copy_to_clipboard);
         }
     }
-
-    @Subscribe(threadMode = ThreadMode.MAIN)
-    public void onMediaChanged(MediaStore.OnMediaChanged event) {
```

Comment:
Fixed in 603516462c8475c3437ff9d057e73608a0a39e1c.

I forgot about how that worked. I think it's better to remove the dispatcher from places where we don't use it.